keyboards/southpole/keymaps/default/keymap.c

```diff
@@ -0,0 +1,25 @@
+#include "southpole.h"
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	KEYMAP(
```

Comment:
Please change this to:

```suggestion
	LAYOUT(
```